extensions/ql-vscode/src/view/common/__tests__/Alert.spec.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import { render, screen } from '@testing-library/react';
+import { Alert } from '../Alert';
+
+describe(Alert.name, () => {
+  it('renders a warning correctly', () => {
+    render(<Alert type="warning" title="Warning title" message="Warning content" />);
+
+    expect(screen.getByText('warning: Warning title')).toBeInTheDocument();
```

Comment:
It is indeed uppercased by the styling, but you're right that when you copy the text it would show as `warning`. Perhaps it would be even better to use `Warning`? That would make the most sense when copying this text, such that is shows as e.g. `Warning: Access mismatch`.

Reply:
I agree that capitalisation would be the ideal 👍